azurelinuxagent/ga/exthandlers.py

```diff
@@ -81,13 +85,12 @@
 _NUM_OF_STATUS_FILE_RETRIES = 5
 _STATUS_FILE_RETRY_DELAY = 2  # seconds
 
+# This is the default sequence number we use when there are no settings available for Handlers
+_DEFAULT_SEQ_NO = "0"
 _ENABLE_EXTENSION_TELEMETRY_PIPELINE = True
```

Comment:
This variable is not needed anymore, the logic was moved to `agent_supported_features.py`. It can be removed

Reply:
thanks; removed